keyboards/handwired/p1800fl/config.h

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2020 marhalloweenvt
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0xFEED
+#define PRODUCT_ID 0x0000
```

Comment:
```suggestion
#define PRODUCT_ID 0x3EAE
```

```
echo -n "p1800fl" | shasum
```

Reply:
Sorry, but what does this means

```
echo -n "p1800fl" | shasum
3eae5018ebc673f11753103673976a7f4f6ca851  -
```

Do i need to adjust anything for this?

Reply:
This is just how we generate a pseudo-unique PID when the VID is `0xFEED`. We use the first four digits of the SHA1 hash of the product name without the newline character.